base/cvd/cuttlefish/host/libs/image_aggregator/image_aggregator.cc

```diff
@@ -294,6 +295,31 @@ Result<void> DeAndroidSparse(const std::vector<ImagePartition>& partitions) {
   return {};
 }
 
+Result<CompositeDisk> CompositeDiskFromFile(const std::string& path) {
```

Comment:
This is redundant with [`CompositeDiskImage::OpenExisting`](https://github.com/google/android-cuttlefish/blob/4a1c13260a86def7c32280f68a674f3ff22f9062/base/cvd/cuttlefish/host/libs/image_aggregator/composite_disk.cc#L33), though the `CompositeDiskImage` class is currently missing accessors to get the proto type.